Description: Remove the hard-to-satisfy python dependency on dbus-python

Add the dependency at the OS package level.

--- a/setup.py
+++ b/setup.py
@@ -91,7 +91,6 @@
     install_requires=[
         "cachetools",
         "jinja2",
-        "dbus-python",
         "pygobject",
         "flask",
         "python-networkmanager",
